mono/profiler/log.c

```diff
@@ -1656,6 +1659,37 @@ finalize_object_end (MonoProfiler *prof, MonoObject *obj)
 	EXIT_LOG;
 }
 
+static void
+fileio (MonoProfiler *prof, uint64_t kind, uint64_t size)
```

Comment:
Not sure we need to update log profiler to handle this, as that's just a sample profiler. But changes don't hurt.